azure/notebooks/Train-RAPIDS.ipynb

```diff
@@ -280,11 +271,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#rapids-cloud-ml image is available in Docker Hub\n",
-    "image_name = 'rapidsai/rapidsai-cloud-ml:0.18-cuda11.0-base-ubuntu18.04-py3.8'\n",
+    "from azureml.core import Environment\n",
+    "\n",
+    "#environment file\n",
+    "environment_file = \"Dockerfile\"\n",
```

Comment:
What does this Dockerfie correspond to? Is there supposed to be a dockerfile included in the directory?

Reply:
Yup created one so you can modify for your example, just pushed that as well